src/cdk/v2/destinations/bluecore/utils.js

```
@@ -152,6 +158,13 @@ const isStandardBluecoreEvent = (eventName) => {
   return !!EVENT_NAME_MAPPING.some((item) => item.dest.includes(eventName));
 };
 
+const isSubscriptionEvent = (eventName) => {
+  if (typeof eventName !== 'string') {
+    return false;
```

eventName should always non-empty string. If this condition fails, then there is some critical plaform issue. Maybe we should throw a platform error here?